### ISSUE TITLE
Screencapture not working on mac os.

### ISSUE THREAD

**Issue by roylkng** [LucianoGanga/simple-headless-chrome#29]
Captures an empty image of 10kb, i can see the html elements in the logs though.

**Comment by hparadiz** [comment]
Try this. Worked for me

```js
await browser.saveScreenshot('test',{
    fromSurface: true
});
```

**Comment by LucianoGanga** [comment]
Hi @roylkng ! Can you submit your code so we can help you better? 

Thanks! 
Luciano

**Comment by roylkng** [comment]
```
const HeadlessChrome = require('simple-headless-chrome')

var browser = new HeadlessChrome({
  headless: true // If you turn this off, you can actually see the browser navigate with your instructions
  // see above if using remote interface
})
async function navigateWebsite(file) {
  console.log(file.length)
  await browser.goTo('https://idatalabs.com/tech/products/autodesk-inventor')
  // await browser.wait(180)
  const htmlTag = await browser.evaluate(function(selector) {
    const selectorHtml = document.querySelector(selector)
    return selectorHtml.innerHTML
  }, '.market-share-chart-container');
  console.log(htmlTag)
// tried without fromSurface paramaeter
  await browser.saveScreenshot('./' + file[0].filename, {quality: 100, fromSurface: true})
  console.log("created Image")
  file.shift();
  return file
 }
```

**Comment by LucianoGanga** [comment]
@roylkng sorry for the late reply. I'll be working on this during this week. I had a surgery and I was online for a few days. 

As soon as I can make it work again, I'll upload the fix and let you know! 

Cheers, 
Lucho

**Comment by LucianoGanga** [comment]
I had to rewrite the screenshot methods, as the Chrome DevTools Protocol changed a little bit and deprecated some methods. So this is available since v4.3.0 :) 

Here you can find a working example of the screenshots. 

https://github.com/LucianoGanga/simple-headless-chrome/blob/master/examples/screenCapture.js

I'll close the issue, but feel free to reopen it if something goes wrong. 

Cheers!
Lucho
